### DISCUSSION TITLE
Expectativas 2019

### DISCUSSION THREAD

**Discussion by ViniciusGularte** [frontendbr/forum#1328] [Discussão]
Pra começar um feliz natal a todos desenvolvedores e desenvolvedoras que mantém esse grupo ativo, vocês são fodas :heart: :tada: 

Quais são suas expectativas para esse novo ano no front ? O que vai bombar e o que vai cair ?

Já desejando um feliz ano novo a todos de antemão :smile: :tada:

**Comment by luizbills** [comment]
Não me importo com o que vai bombar... só espero que o brasil não caia mais kkk

**Comment by SrVieira** [comment]
Olha, eu tenho uma fé que a área de **front** e **UX** serão umas das que terão um maior crescimento. Algumas pesquisas já realizadas, seja no mercado nacional ou externo apontam para isto. Em relação a tecnologias acredito que o React continuará em crescimento, e fora deste contexto uma ótima tecnologia a focar é o GraphQL.

E como o amigo acima falou, esperamos que o Brasil não caia

**Comment by AmandaFonseca** [comment]
@SrVieira Gostei desse lance do GraphQL eu consigo usar ele no Phyton?

**Comment by williangd** [comment]
@AmandaFonseca consegue sim. https://github.com/graphql-python/graphene

**Comment by AmandaFonseca** [comment]
@williangd Obrigada ^.^

**Comment by Alexsandra127** [comment]
Estou começando agora na área. Espero que front end ganhe mais destaque em 2019. Feliz ano novo pra todos! 😺🎉🎊

**Comment by has256** [comment]
Feliz ano novo galera! :tada:

Assim como a @Alexsandra127 também estou no comecinho da jornada, espero que pra mim, seja um ano de muito aprendizado e de **muito** Vanilla JS, hahahah.   

Em relação à tecnologias: também acho que o React e o GraphQL continuarão crescendo. Além disso, também imagino que o TypeScript siga o mesmo caminho (talvez o ReasonML também). Outras tecnologias que pretendo colocar no meu radar pro próximo ano são: GatsbyJS e Next.js. Também suspeito que talvez apareça algum novo player pra tretar com vue/angular/react.

...e que o Brasil não caia + (ou num cenário ainda pior, exploda).

**Comment by claudiobernardo** [comment]
Iniciarei meus estudos de Front-End em 2019, espero aprender muito aqui no fórum com vocês!

**Comment by isacmoura** [comment]
Vejo uma ascensão de GraphQL cada vez maior. Além disso, ouvi bastante falar sobre micro frontends para o próximo ano, está na minha pauta de estudos.

**Comment by vitor-mendrone-deel** [comment]
Esse post [A Recap of Frontend Development in 2018](https://levelup.gitconnected.com/a-recap-of-frontend-development-in-2018-715724c9441d) faz algumas previsões sobre o ano de 2019 e eu concordo com todas. O @LFeh traduziu essas  previsões em uma [thread no Twitter](https://twitter.com/LFeh/status/1072838228267462657).

É um bom momento pra começar na área, o mercado está aquecido, as tecnologias estão passando por uma estabilização e a tendência é que isso se estenda nos próximos 2 ou 3 anos. 

Bom 2019, galera!

**Comment by leocavalcante** [comment]
Olá pessoal, sobre o GraphQL, eu tinha aberto essa issue aqui há um tempo atrás https://github.com/frontendbr/forum/issues/191 Acho que na época ainda não era tão popular rs Até fechei, mas abriram de novo, fica o headsup 😃

**Comment by didiraja** [comment]
Minha obrigação esse ano é terminar bom em JS e confortável com React

**Comment by felipefialho** [comment]
Para quem se interessar, [Como foi o Desenvolvimento Front-end em 2018 e o que esperar para 2019](https://medium.com/@lfeh/como-foi-o-desenvolvimento-front-end-em-2018-e-o-que-esperar-para-2019-c64a55854bc3).

Vale a leitura.

**Comment by hmilena** [comment]
Boa @didiraja. Tô com essa mesma expectativa.
